scripts/benchmark.js

```diff
@@ -14,126 +14,181 @@ const fs     = require("fs");
 const path   = require("path");
 const compile = require("google-closure-compiler-js").compile;
 
-const filename = process.argv[2];
-if (!filename) {
-  console.error("Error: No filename specified");
-  process.exit(1);
+const packagename = process.argv[2];
+var filename = process.argv[3];
+
+if (!packagename) {
+	console.error("Error: No package specified");
+	process.exit(1);
 }
 
+console.log("package: " + packagename);
+
+const pathToScripts = "./scripts";
+
 const table = new Table({
-  head: ["", "raw", "raw win", "gzip", "gzip win", "parse time", "run"],
```

Comment:
As a maintainer I can't imagine anything more likely to make me reject the change than a load of whitespace changes! It's far easier to review when you stick with the conventions they're already using.